db/pagination/pagination_test.go

```diff
@@ -29,12 +29,12 @@ func TestNewPaginator(t *testing.T) {
 	Expect(err).ToNot(HaveOccurred())
 }
 
-func TestNewPaginatorDefaults(t *testing.T) {
+func TestNewPaginatorNoDefaults(t *testing.T) {
 	RegisterTestingT(t)
 	pg, err := NewPaginator(nil, "", "", 0, 0)
 	Expect(err).ToNot(HaveOccurred())
-	Expect(pg.Key).To(Equal(defaultSortKey))
-	Expect(pg.Order).To(Equal(ASC))
+	Expect(pg.Key).To(Equal(""))
```

Comment:
No, please check all usages of this function. Backward compatibility is preserved.